src/main/java/uk/gov/justice/digital/common/CommonDataFields.java

```diff
@@ -42,4 +44,13 @@ public static StructType withMetadataFields(StructType schema) {
                 .add(DataTypes.createStructField(OPERATION, DataTypes.StringType, false))
                 .add(DataTypes.createStructField(TIMESTAMP, DataTypes.StringType, false));
     }
+
+    /**
+     * Add the SCD fields to the provided schema
+     */
+    public static StructType withScdFields(StructType schema) {
```

Comment:
Adds the columns checkpoint_col and update_type to the in-memory schema to allow successful schema validation. This is required because the raw data being validated now contains both columns